src/unittests/f32_interval.spec.ts

```diff
@@ -1129,3 +1140,61 @@ g.test('atan2Interval')
       `atan2Interval([${x}, ${y}]) returned ${got}. Expected ${expected}`
     );
   });
+
+g.test('multiplicationInterval')
+  .paramsSubcasesOnly<BinaryToIntervalCase>(
+    // prettier-ignore
+    [
+      // 32-bit normals
+      { input: [0, 0], expected: [0, 0] },
+      { input: [1, 0], expected: [0, 0] },
+      { input: [0, 1], expected: [0, 0] },
+      { input: [-1, 0], expected: [0, 0] },
+      { input: [0, -1], expected: [0, 0] },
+      { input: [1, 1], expected: [1, 1] },
+      { input: [1, -1], expected: [-1, -1] },
+      { input: [-1, 1], expected: [-1, -1] },
+      { input: [-1, -1], expected: [1, 1] },
+      { input: [2, 1], expected: [2, 2] },
+      { input: [1, -2], expected: [-2, -2] },
+      { input: [-2, 1], expected: [-2, -2] },
+      { input: [-2, -1], expected: [2, 2] },
+      { input: [2, 2], expected: [4, 4] },
+      { input: [2, -2], expected: [-4, -4] },
+      { input: [-2, 2], expected: [-4, -4] },
+      { input: [-2, -2], expected: [4, 4] },
```

Comment:
Please check that max-normal times max-normal produces infinity.
And negative max-normal times max-normal produces negative infinity.

Reply:
Done